pkg/app/pipedv1/plugin/kubernetes/deployment/server.go

```diff
@@ -33,10 +35,20 @@ import (
 	"google.golang.org/grpc/status"
 )
 
+const (
+	defaultKubectlVersion = "1.18.2"
```

Comment:
This is copied from current implementation.
https://github.com/pipe-cd/pipecd/blob/40dae444572e9073bdc68e20d13c24c1911b64a5/pkg/app/piped/toolregistry/install.go#L29